### PR TITLE
Address Stage 3 review feedback

### PR DESCRIPTION
**Pull request by leobalter**
Ref #17

### CODE REVIEW

File: spec.emu [hunk 1]
```diff
@@ -15,14 +15,16 @@ contributors:
   - Rick Waldron
 </pre>
 
-<emu-clause id="sec-hasidentity" aoid="HasIdentity">
-  <h1>HasIdentity ( _argument_ )</h1>
-  <p>The abstract operation HasIdentity takes argument _argument_ (an ECMAScript language value). It determines if _argument_ is considered to have a unique identity, enabling _argument_ to be a key of a WeakMap or entry in a WeakSet. This operation performs the following steps when called:</p>
+<emu-clause id="sec-isobjectorsymbol" aoid="IsObjectOrSymbol">
+  <h1>IsObjectOrSymbol ( _argument_ )</h1>
+  <p>The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value) and determines if its Type is Object or Symbol. This operation performs the following steps when called:</p>
```
Comment (leobalter) [inline spec.emu:20]: If we want to match styles for consistency, here are some options:

[IsArray](https://tc39.es/ecma262/#sec-isarray), [IsRegExp](https://tc39.es/ecma262/#sec-isregexp):

> The abstract operation IsObjectOrSymbol takes argument _argument_. It performs the following steps when called:

[IsCallable](https://tc39.es/ecma262/#sec-iscallable):

> The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value). It determines if _argument_ is an object or a symbol. It performs the following steps when called:

---

There are inconsistencies in ECMA-262 using "an Object" or "an object", I honestly just prefer what is better accepted by the editors.

**Review by ljharb — APPROVED** [review]
While I’d prefer a name that describes what it means instead of what it is, this is better than “identity”

**Comment by leobalter** [comment]
I want to continue investigating a better name for this, but I want to not rush it before the meeting next week to then pick a good one.

**Comment by nicolo-ribaudo** [comment]
> While I’d prefer a name that describes what it means instead of what it is, this is better than “identity”.

While reading the spec text I disliked `IsObjectOrSymbol` because it seems like an "implementation detail" of that AO rather than saying where it needs to be used. It's possible that in the future we will want to allow R&T containing symbols as WM keys, and we would have to rename the AO because it doesn't describe the pourpouse.

Then I found this comment and I agree :smile: 

WDYT about one of `IsWeaklyHoldable`, `CanHoldWeakly`, or `CanReferenceWeakly`?

**Comment by ljharb** [comment]
IsWeakable :-p

**Pull request merged**